x-pack/plugins/cases/server/telemetry/queries/cases.ts

```diff
@@ -52,17 +57,94 @@ export const getLatestCasesDates = async ({
   ]);
 
   return {
-    createdAt: savedObjects?.[0].saved_objects?.[0].attributes?.created_at ?? null,
-    updatedAt: savedObjects?.[1].saved_objects?.[0].attributes?.updated_at ?? null,
-    closedAt: savedObjects?.[2].saved_objects?.[0].attributes?.closed_at ?? null,
+    createdAt: savedObjects?.[0]?.saved_objects?.[0]?.attributes?.created_at ?? '',
```

Comment:
Prior to adding the `?` this was causing an exception if `saved_objects[0]` was undefined which was skipping returning `null`. Now that this doesn't throw an error I'm getting test failures because a keyword field can't be mapped to null 🤷‍♂️ .